ExaFunctions/CsvBulkImportFromBlobStorage.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Text;
+using Exasol.Utilities;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace Exasol
+{
+    public static class CsvBulkImportFromBlobStorage
+    {
+        [FunctionName("CsvBulkImportFromBlobStorage")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "post", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            try
+            {
+                log.LogInformation("C# HTTP trigger function processed a request.");
+
+                string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
+                dynamic data = RequestUtilities.DeserializeJsonBody(requestBody);
+
+                CheckParameters(data);
+                //POST body
+                //the target db connection string
+                string dbConnectionString = data?.dbconnectionstring;
+                //the target db table
+                string dbTable = data?.dbtable;
+                //the azure storage account connection string
+                string azureStorageAccountConnectionString = data?.storageaccountconnectionstring;
+                //the name of the blob storage container withing the storage account
+                string azureStorageAccountContainerName = data?.storageaccountcontainername;
+                //the filepath within the blob storage container
```

Comment:
I'm not sure you need comments here. I think variables names are quite descriptive and comments are somehow duplication

Reply:
I'm glad you think the variable names are well chosen. I could remove them but I feel it doesn't really hurt in this case?.

Reply:
I think it can hurt because you basically need to maintain two things: comments and the variable names. If you rename a variable, you also need to re-write the comment. If you forget to re-write the comment - the comment becomes misleading. So I strongly recommend avoiding comments when possible